downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,353 @@
+  | `getExpressionStringValue(Object, BLink)`        | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `parseExpressionStringValue(String, BLink)`      | `io.ballerina.runtime.api.utils.StringUtils`       |
+  | `getValueKind()`                                 | `io.ballerina.runtime.api.values.BDecimal`         |
+  | `getStrand()`                                    | `io.ballerina.runtime.api.values.BFuture`          |
+  | `call(Strand, String, Object...)`                | `io.ballerina.runtime.api.values.BObject`          |
+  | `start(Strand, String, Object...)`               | `io.ballerina.runtime.api.values.BObject`          |
+  | `getRegExpDisjunction()`                         | `io.ballerina.runtime.api.values.BRegexpValue`     |
+  | `instantiate(Strand)`                            | `io.ballerina.runtime.api.values.BTypedesc`        |
+  | `instantiate(Strand, BInitialValueEntry[])`      | `io.ballerina.runtime.api.values.BTypedesc`        |
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.6.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
+
+### Deprecations
+
+#### `stan` package
+
+- Deprecated the `stan` package.
+    > For NATS-enabled applications requiring persistence, it is recommended to use the `JetStreamClient` provided by the [`ballerinax/nats`](https://github.com/ballerina-platform/module-ballerinax-nats) library. For more information, see the new [NATS JetStream client](https://lib.ballerina.io/ballerinax/nats/latest#JetStreamClient) and the [NATS JetStream listener](https://lib.ballerina.io/ballerinax/nats/latest#JetStreamListener).
+
+#### `serdes` package
+
+- Deprecated the `serdes` package.
+
+### New features
+
+#### `constraint` package
+
+- Introduced the `@constriant:Date` annotation to validate date record structures.
+- Allowed constraint annotations on subtypes.
+
+#### `http` package
+
+- Added constraint validation support for `query`, `path`, and `header` parameters.
+- Exposed the `http:Request` object in the response interceptors as a remote method parameter.
+- Added finite type support for `query`, `path`, and `header` parameters.
+
+#### `graphql` package
+
+- Added support for GraphQL field interceptors.
+- Added support for GraphQL interceptor configurations.
+- Added support to access subfields of a GraphQL field from the `graphql:Field` object.
+
+#### `persist` package
+
+- Added support for the `in-memory` data store.
+- Added support for the `google-sheets` data store. This is currently an experimental feature and its behavior may be subject to change in future releases.
+- Added support for defining enums and adding `enum` fields to the data model.
+
+#### `edi` package
+
+- Introduced the `edi` module to convert EDI to JSON and vice versa based on a given schema.
+
+#### `toml` package
```

Comment:
We better mention toml and yaml are in pre-release versions . @shafreenAnfar / @MadhukaHarith92

Reply:
Fixed in the below.

- https://github.com/ballerina-platform/ballerina-dev-website/pull/6975#discussion_r1209037423
- https://github.com/ballerina-platform/ballerina-dev-website/pull/6975#discussion_r1209037684